projects/plugins/jetpack/extensions/blocks/mailchimp/loader.js

```diff
@@ -0,0 +1,18 @@
+import { Placeholder, Spinner } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { BLOCK_CLASS } from './constants';
+
+const Loader = ( { icon, notices } ) => (
+	<Placeholder
+		icon={ icon }
+		notices={ notices }
+		className={ BLOCK_CLASS }
+		label={ __( 'Mailchimp', 'jetpack' ) }
```

Comment:
I know this was already in previous version of code, but I don't think we need to send brand names to translators.